_includes/1.0/api/customer.html

```diff
@@ -238,6 +238,32 @@ <h4>Individual only</h4>
     <h3 id="customerjsonexample">JSON Example</h3>
     {% highlight bash %}{% include 1.0/api/json/customer.json %}{% endhighlight %}
 
+    <h3 id="customerfilter">Available Filter Fields</h3>
+    <table class="table table-bordered table-striped">
+        <thead>
+        {% include table-header-filter.html %}
+        </thead>
+        <tbody>
+        <tr>
+            <td>assigned_to</td>
+            <td>Filter by employee that is assigned to the customer.</td>
```

Comment:
Can you add '...employee id that is...', to make it clear that the expected parameter is the id.

Maybe also add (e.g assigned_to=3) too.